python/hsfs/feature_view.py

```diff
@@ -2476,6 +2478,11 @@ def _create_statistics_monitoring(
                 Options are "HOURLY", "DAILY", "WEEKLY", "MONTHLY", defaults to "DAILY".
```

Comment:
Same as previous comment.
In the JobScheduler constructor, it seems MONTHLY it is not supported, only HOURLY, DAILY and WEEKLY.